core/services/runs.go

```diff
@@ -343,6 +343,11 @@ func updateTaskRunConfirmations(currentHeight *models.Big, jr *models.JobRun, ta
 	}
 
 	confs := blockConfirmations(currentHeight, jr.CreationHeight)
+	if confs.Cmp(big.NewInt(0)) <= 0 { // negative, so floor at 0
```

Comment:
I don’t believe it makes a difference, since the following line takes the minimum of two values, and with uint that’s 0. I asked myself the same thing though, so I will change it for better legibility.